cmd/kube-apiserver/app/server.go

```diff
@@ -430,6 +430,7 @@ func BuildGenericConfig(s *options.ServerRunOptions) (*genericapiserver.Config,
 			versionedInformers.Core().V1().Services().Lister(),
 		)
 	}
+	genericConfig.SharedInformerFactory = versionedInformers
```

Comment:
just wondered whether anything failed or panic'ed without the versionedInformers.